### PR TITLE
Material You design changes and widgets update (Part 2)

### PR DESCRIPTION
**Pull request by TipzRickyCheung**

### CODE REVIEW

**Comment by TipzRickyCheung** [comment]
89a6fdb26ab4c94724bb3e2b13ce493fe8e28eb4 will enable the small text. But I am not sure whether the small text is preferred, or if we like it showing the battery percentage directly.
It seems to differ with versions, the behaviour I mean. It shows the small text without the change on Android 12, but not on Android 14.

**Comment by d4rken** [comment]
>  But I am not sure whether the small text is preferred, or if we like it showing the battery percentage directly.

I prefer the %, the text is a nice info, but the % is the main info everyone is looking for, what do you think?

File: app/src/main/java/eu/darken/capod/monitor/ui/MonitorNotifications.kt [hunk 1]
```diff
@@ -80,31 +79,30 @@ class MonitorNotifications @Inject constructor(
             // Options here should be mutually exclusive, and are prioritized by their order of importance
             // Some options are omitted here, as they will conflict with other options
             // TODO: Implement a settings pane to allow user to customize this
-            device.apply {
+            when (device) {
```
Comment (d4rken) [inline app/src/main/java/eu/darken/capod/monitor/ui/MonitorNotifications.kt:82]: Nice variant! My style would have been something like this, but everyone writes their code a little different :heart: 

```kotlin
@StringRes val contentTitleRes = when {
                // Pods charging state
                // This goes first as pods should not be worn if it is still charging
                device is HasChargeDetection && device.isHeadsetBeingCharged -> {
                    eu.darken.capod.common.R.string.pods_charging_label
                }

                // Pods wear state
                device is HasEarDetection -> when {
                    device.isBeingWorn -> eu.darken.capod.common.R.string.headset_being_worn_label
                    else -> eu.darken.capod.common.R.string.headset_not_being_worn_label
                }

                // Case charge state
                // This is under pods wear state as we don't want it conflicting with it
                device is HasCase && device.isCaseCharging -> {
                    eu.darken.capod.common.R.string.pods_charging_label
                }
                
                else -> eu.darken.capod.common.R.string.pods_case_unknown_state
            }
```

Comment (TipzRickyCheung) [reply app/src/main/java/eu/darken/capod/monitor/ui/MonitorNotifications.kt:82]: Seems like I am still not accustomed to the Kotlin syntax 😅

**Review by d4rken — APPROVED** [review]

**Pull request merged**